matcher/initialise_standard_univ_list.py

```diff
@@ -10,19 +19,14 @@ def initialise_standard_univ_list():
     all_univs = None
     acronyms = None
 
-    if not os.path.exists('data/GridUnivsData/grid.csv'):
-        all_univs = pd.read_csv(f"{path}{sub_path}grid.csv")
-        print("Fetch from remote Grid")
-    else:
-        all_univs = pd.read_csv('data/GridUnivsData/grid.csv')
-    # Using acronyms to make the fuzzy matcher acronym sensitive as well
-    if not os.path.exists('data/GridUnivsData/acronyms.csv'):
-        acronyms = pd.read_csv(f"{path}{sub_path}acronyms.csv")
-        print("Fetch from remote Acronym")
+    all_univs = download_file_if_unavailable(path, sub_path, 'data/GridUnivsData/grid.csv')
+    acronyms = download_file_if_unavailable(path, sub_path, 'data/GridUnivsData/acronyms.csv')
+    aliases = download_file_if_unavailable(path, sub_path, 'data/GridUnivsData/aliases.csv')
 
-    else:
-        acronyms = pd.read_csv('data/GridUnivsData/acronyms.csv')
+    aliases = aliases.rename(columns={"grid_id": "ID"})
+
+    standard_and_alias_univ_names = pd.merge(all_univs, aliases, on='ID', how='outer')
```

Comment:
Is it an outer join what we want? Seems like a left join is the correct type, but I remember we have fixed that before...